python/paddle/fluid/io.py

```diff
@@ -323,9 +326,10 @@ def save_inference_model(dirname,
     if isinstance(feeded_var_names, basestring):
         feeded_var_names = [feeded_var_names]
     else:
-        if not (bool(feeded_var_names) and all(
-                isinstance(name, basestring) for name in feeded_var_names)):
-            raise ValueError("'feed_var_names' should be a list of str.")
+        if len(feeded_var_names) > 0:
```

Comment:
Should we throw error if `feeded_var_names` is not an array? Also I found the correct spell is `fed_var_names`, not `feeded_`. And it is being used in many places.

Reply:
oh, feeded wrong, can you help to correct it. I think `feed_var_names` is ok.